File: Classwork/OOP/account-exception-app/src/com/monocept/model/test/AccountTest.java
/**Copyright to customer-app by asif
All rights reserved.
*/
package com.monocept.model.test;

import com.monocept.model.Account;

public class AccountTest {

	public static void main(String[] args) {
		Account acc1 = new Account(101, "abc", 1000);
		try {
			acc1.withdraw(1200);

		} catch (RuntimeException ex) {
			System.out.println(ex.getMessage());
//			System.out.println("withdraw amount is greater than minimum balance....");

		} finally {
			printDetails(acc1);
//			System.out.println("withdraw amount is greater than minimum balance....");

		}
	}

	private static void printDetails(Account acc1) {
		System.out.println();
		System.out.println("Account number is : " + acc1.getaccno());
		System.out.println("Name is :" + acc1.getname());
		System.out.println("Balance is : " + acc1.getbalance());
	}
}

File: nodejs_demos/custom_modules/Account_Test.js
const Account = require("./Account");
const acc1 = new Account(100,"rahul",1000);
const acc2 = new Account(101,"sohail",5000);
const acc3 = new Account(102,"raj",6000);


acc1.deposit(1000);

getDetails = function(){
    console.log("Id is"  +acc1.getId());
    console.log("Name is"  +acc1.getName());
    console.log("Balance is"  + " "+ acc1.getBalance());
}
getDetails();
